data_pipelines/rne/database/process_rne.py

```diff
@@ -251,16 +255,13 @@ def extract_dirigeants_data(entity, file_type="flux"):
     list_dirigeants_pp = []
     list_dirigeants_pm = []
 
-    if file_type == "flux":
-        siren, date_maj, dirigeants = get_company_data_from_flux(entity)
-
-    elif file_type == "stock":
-        siren, date_maj, dirigeants = get_company_data_from_stock(entity)
+    siren, date_maj, dirigeants = get_dirigeants_from_entity(entity, file_type)
 
     for dirigeant in dirigeants:
-        type_de_personne = dirigeant.get("typeDePersonne", None)
+        type_of_personne = dirigeant.get("typeDePersonne", None)
+        is_personne_physique = True if dirigeant.get("entrepreneur", {}) else False
```

Comment:
this loop could be more readable

- you could merge `type_of_personne` and is `personne_physique` r
- you could move every creation of diriegant_pp in a dedicated helper
- what happens if all conditions fail ?

Reply:
yeah yeah yeah 😆